### PR TITLE
feat: set real data to `totalFailureAndSeverity` widget

### PR DESCRIPTION
**Pull request by yuda110**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [ ] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [x] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
SRD

스크린샷 2023-06-12 오후 10 50 50

### CODE REVIEW

File: apps/web/src/services/dashboards/widgets/asset-widgets/total-failure-and-severity/TotalFailureAndSeverityWidget.vue [hunk 1]
```diff
@@ -255,22 +224,50 @@ const state = reactive({
...
+        return [];
+    }
+};
+const fetchRealtimeData = async (): Promise<CloudServiceStatsModel[]> => {
+    try {
+        apiQueryHelper
+            .setFilters(state.consoleFilters)
+            .addFilter({ k: 'ref_cloud_service_type.labels', v: 'Compliance', o: '=' })
+            .addFilter({ k: 'key', v: ['fail_finding_count', 'pass_findings_count'], o: '' });
+        const prevMonth = dayjs.utc(state.settings?.date_range?.start).subtract(1, 'month').format(DATE_FORMAT);
```
Comment (WANZARGEN) [inline apps/web/src/services/dashboards/widgets/asset-widgets/total-failure-and-severity/TotalFailureAndSeverityWidget.vue:261]: issue: I think `start` must be changed to `end`!

**Review by WANZARGEN — APPROVED** [review]
What a complex requirement! 👍
I left a comment that looks a critical issue!

File: apps/web/src/services/dashboards/widgets/asset-widgets/total-failure-and-severity/TotalFailureAndSeverityWidget.vue (hunk 1 above)
Comment (yuda110) [reply apps/web/src/services/dashboards/widgets/asset-widgets/total-failure-and-severity/TotalFailureAndSeverityWidget.vue:261]: Sooooo thank you😭

**Review by sulmoJ — APPROVED** [review]
LGTM.

**Pull request merged**